rmf_demos_fleet_adapter/rmf_demos_fleet_adapter/RobotCommandHandle.py

```diff
@@ -506,6 +510,7 @@ def projection(current_position,
         # Spin on the spot
         if approach_lanes is None or len(approach_lanes) == 0:
             return None
+        approach_lane_limit = 0.0
```

Comment:
Initializing this to `0.0` would lead to an incorrect output. The `appraoch_lane_limit = min(appraoch_lane_limit, lane_limit)` will always return `0.0`. So what you want to do is initialize this to something like `np.inf`. Then when you iterate through the appraoch lanes, if there is a speed limit defined and if it is smaller than `appraoch_lane_limit` then you set `appraoch_lane_limit` to that value. Then the functions returns `0.0` if `appraoch_lane_limit  == np.inf`, else return `appraoch_lane_limit`.

Reply:
There is a check for whether `approach_lane_limit` is more than 0 [here](https://github.com/xiyuoh/rmf_demos/blob/add_speed_limit/rmf_demos_fleet_adapter/rmf_demos_fleet_adapter/RobotCommandHandle.py#L525), implemented the same in the latest commit, but I'm ok to change to `np.inf` if that's preferred